.github/workflows/unit-test.yml

```
@@ -235,19 +227,14 @@ jobs:
         steps:
             - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
 
-            - name: Use desired version of NodeJS
-              uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
-              with:
-                  node-version-file: '.nvmrc'
-                  cache: npm
+            - name: Setup Node.js and install dependencies
+              uses: ./.github/setup-node
 
-            - name: Npm install and build
+            - name: Npm build
```

Hmm, this seems to be running the Lerna build script. 
```suggestion
            - name: Build Lerna
```

---

"Build Lerna` isn't really descriptive either though. I think this should be "Build packages" and just run `npm run build:packages`. But it's already the way it is in trunk, so maybe we should change it in another PR?